vfs/vfscommon/path.go

```diff
@@ -5,6 +5,9 @@ import (
 	"path/filepath"
 )
 
+// Constants
+const LinkSuffix = ".rclonelink" // The suffix added to a translated symbolic link
```

Comment:
This should probably go in `fs` as `vfs` is a higher layer above the backends and importing it from a backend looks weird!